### ISSUE TITLE
Can't scroll

### ISSUE THREAD

**Issue by schwarzen13** [jaredrummler/MaterialSpinner#79]
Hello, I'm using your Spinner for my company purposes (non commercial). Thank you for your absolutely beautiful spinner. The main problem is, I have so many data to show, but the spinner content can't be scrolled. Can you please help me? Thanks.

**Comment by rodrigordgfs** [comment]
## I have the same problem, if anyone knows how to fix it, please, help us!
### Image of my Activity
[https://imgur.com/a/qX3yf](url)

**Comment by jaredrummler** [comment]
I'm not sure what you guys are referring to. The spinner items are scrollable. Example:

```java
MaterialSpinner spinner = (MaterialSpinner) findViewById(R.id.spinner);
String[] beer = new String[100];
for (int i = 0; i < beer.length;) {
  beer[i] = String.format("%d beer on the wall", ++i);
}
spinner.setItems(beer);
```

![Imgur](https://i.imgur.com/lMwfLNJ.gif)

**Comment by pzienowicz** [comment]
The same issue:
![screenshot_20180215-224325](https://user-images.githubusercontent.com/3447652/36282560-d606f92c-12a1-11e8-9ac6-b7895ef13ba0.png)

I am using MaterialSpinnerAdapter.

**Comment by rodrigordgfs** [comment]
@jaredrummler I'm using the same code, but doesn't work, I will try again

**Comment by pzienowicz** [comment]
@rodrigordgfs @schwarzen13 ,
below code solves our problems:
`app:ms_dropdown_max_height="350dp"
 app:ms_dropdown_height="wrap_content"`

**Comment by Thoimanich** [comment]
same here

**Comment by toledorobson** [comment]
@pzienowicz thank you, it solved.

**Comment by adelariv** [comment]
@pzienowicz  was really helpful for me, Thanks! ;)

**Comment by pzienowicz** [comment]
Is solves our problem, but not not completely.
I don't want to hardcode max height of dropdown list, because it should be calculated automatically (depends on screen orientation, etc.). 

@jaredrummler could you fix it?
Do you need any code example?

**Comment by edof** [comment]
@pzienowicz thanks for your help :)

**Comment by iamabhinavjain** [comment]
If anyone is still having this problem, a possible workaround is switching to version 1.1.0

**Comment by pzienowicz** [comment]
@iamabhinavjain - yes, you are right.

**Comment by jaredrummler** [comment]
fixed a4c3128a947fc5a4eed4607269b7b3060fb5139e
